shadowsocks-csharp/View/MenuViewController.cs

```diff
@@ -322,6 +320,7 @@ private void LoadMenu()
                     CreateMenuItem("Show Logs...", new EventHandler(this.ShowLogItem_Click)),
                     this.VerboseLoggingToggleItem = CreateMenuItem( "Verbose Logging", new EventHandler(this.VerboseLoggingToggleItem_Click) ),
                     this.ShowPluginOutputToggleItem = CreateMenuItem("Show Plugin Output", new EventHandler(this.ShowPluginOutputToggleItem_Click)),
+                    this.WriteI18NFileItem = CreateMenuItem("Write translation template",new EventHandler(WriteI18NFileItem_Click)),

```

Comment:
Why need this menu? What will happen if user generated an old csv file then user upgrade SS to a new version with new UI elements?

Reply:
> Why need this menu?

user can translate the program by edit generated csv

> What will happen

New translation won't apply

Reply:
> New translation won't apply

IMO, if users need a new language or amend the translation, let them send pull request. Otherwise, by use of an external translation file may cause some "out-of-date" problem after upgrade.

Reply:
If you feel it doesn't matter, please merge the PR.

Reply:
@chenshaoju How do you think?

Reply:
My programming is bad...
Looks good, build succeeded, I think is good.

Reply:
> "out-of-date" problem 

will implement translate file merge to resolve this

Reply:
> will implement translate file merge to resolve this

What will happen if there is a conflict or obsoleted item in new version? This is what I am worrying about. Too much user interactions.
Or your design purpose is to switch between "integrated translation resource" and "external translation file" in the future?

Reply:
> conflict

use external one, as user may changed it

> obsoleted

just keep it, user can downgrade version

Reply:
OK I will draft a new release soon